bigbluebutton-web/grails-app/controllers/org/bigbluebutton/web/controllers/ApiController.groovy

```diff
@@ -123,6 +123,9 @@ class ApiController {
     if(!(validationResponse == null)) {
       invalid(validationResponse.getKey(), validationResponse.getValue())
       return
+    } else if (ParamsUtil.sanitizeString(params.meetingID) != params.meetingID) {
+      invalid("idInvaid", "Meeting ID is invalid")
```

Comment:
```suggestion
      invalid("validationError", "Invalid meeting ID")
```

Using same pattern as in:

https://github.com/bigbluebutton/bigbluebutton/blob/0c951e7f4b64dab12a696ccec79edede8cde04b6/bbb-common-web/src/main/java/org/bigbluebutton/api/model/constraint/MeetingIDConstraint.java#L19

https://github.com/bigbluebutton/bigbluebutton/blob/0c951e7f4b64dab12a696ccec79edede8cde04b6/bbb-common-web/src/main/java/org/bigbluebutton/api/model/constraint/MeetingIDConstraint.java#L20